packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/internal/AppConfigurationImpl.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.mongodb.internal
+
+import io.realm.interop.NativePointer
+import io.realm.interop.RealmInterop
+import io.realm.mongodb.AppConfiguration
+import io.realm.mongodb.AppConfiguration.Companion.DEFAULT_BASE_URL
+import kotlinx.coroutines.CoroutineDispatcher
+
+internal class AppConfigurationImpl(
+    override val appId: String,
+    override val baseUrl: String = DEFAULT_BASE_URL,
+    override val networkTransportDispatcher: CoroutineDispatcher
+) : AppConfiguration {
+
+    private val networkTransport: io.realm.interop.NetworkTransport = KtorNetworkTransport(
+        timeoutMs = 5000,
```

Comment:
`timeoutMs` could be stored in `Constants`? if there's one for the Sync module ?

Reply:
Added a TODO around being able to provide this as a `Duration` through the `AppConfigruation` and define the default there.